local/dfm/lib/utils.sh

```
@@ -0,0 +1,334 @@
+  shift
+  local func="$1"
+  shift
+
+  local cmd_file="${CMD_DIR}/${cmd}.sh"
+  if [[ ! -f "$cmd_file" ]]; then
+    lib::error "Command '$cmd' not found"
+    return 1
+  fi
+
+  # Source the command file
+  source "$cmd_file"
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)